recipe/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "pm4py" %}
+{% set version = "2.7.11.8" %}
```

Comment:
What about the latest version https://github.com/pm4py/pm4py-core/tree/2.7.11.9?

Reply:
We refined this version. The next one has not so many improvements, and they forgot to include `requirements.txt` in the tarball and the build fails. We'll fix it on the next time.

https://inspector.pypi.io/project/pm4py/2.7.11.9/packages/b4/57/f9939756a5fbc4225e4d787387e39009a047b965f12b9f25a627f7cc5299/pm4py-2.7.11.9.tar.gz/

Reply:
Thanks!

Reply:
Thank you!